app/views/posts/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1>Hi from Posts page 📝</h1>
\ No newline at end of file
+<section class="full-post-section">
+  <div class="full-post border-black">
+    <h2>Post</h2>
+    <p class="post-likes-comments">Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
+    <p><%= @post.text %></p>
+  </div>
+  <div class="comments">
+    <%= render "shared/comment", post: @post %>
+  </div>
+</section>
```

Comment:
- [ ] Posts show page should return all the comments, currently your posts show page returns only the last 5 comments

Yours:
![image](https://user-images.githubusercontent.com/62705947/230217643-326d0bfe-f5bf-4044-86ec-b8730c011736.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/230217678-782ede23-c0b6-4601-b1ae-caf812175749.png)